apps/api-frontend/src/app/app.css

```diff
@@ -15,6 +16,16 @@
   display: inline-block;
 }
 
+.app .footer {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.app .footer > div {
+  margin: 1rem;
```

Comment:
This doesn't seem like it's working on the loading page (the two divs are still right next to each other) 
can you give the footer text divs a class name (you can use the same class name for both) and reference them that way?

Reply:
I would use the margin in between the divs and remove the justify-content space-between above 
(space-between can look kind of weird on different devices, and as we're seeing here, doesn't guarantee that the two divs are far apart)